start_bundle_coupled.R

```diff
@@ -650,6 +650,7 @@ for (scen in common) {
 
 if (! "--test" %in% flags && ! "--gamscompile" %in% flags) {
```

Comment:
That should be dropped completely, I think. You should always use `renv`, I think.

Reply:
Done